apps/alpha-test-app/src/App.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { provideI18n } from "sit-onyx";
+import deDE from "sit-onyx/locales/de-DE.json";
+import { useI18n } from "vue-i18n";
+import HelloWorld from "./components/HelloWorld.vue";
+
+const { locale } = useI18n();
+provideI18n({ locale, messages: { "de-DE": deDE } });
```

Comment:
@larsrickert we have to take a look, this gives errors in the IDE that did not happen with the blueprint setup 
<img width="837" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/8ac94396-8def-494b-9c3d-95aabe3ec7fd">

Reply:
solved it by changing the package linking style